plugins/out_pgsql/pgsql.c

```diff
@@ -250,6 +275,9 @@ static void cb_pgsql_flush(struct flb_event_chunk *event_chunk,
     flb_sds_t tag_escaped = NULL;
     size_t str_len;
 
+    if (ctx ==NULL) {
```

Comment:
there's a space missing here

Reply:
fixed in the commit: out_pgsql: code-format &if config err then log err